src/makefiles/linux_atlas_ppc64le.mk

```diff
@@ -1,5 +1,8 @@
 # ATLAS specific Linux ppc64le configuration
 
+ifndef DEBUG_LEVEL
+$(error DEBUG_LEVEL not defined.)
+endif
```

Comment:
Not a show-stopper, but I am wondering why do we carry this (and many similar checks) over so many files. Yes, kaldi.mk better be sane, but on the other hand, all these copy-paste sanity checks are so numerous that they nearly defeat their own purpose.

Reply:
I agree. I just followed existing conventions for the feature work, but yes, this is a clear case where duplicate code ought to be factored out.